Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.44308&attempt_id=c51c541c-1e2a-4e5d-b20d-52577bd4f551)

Attempted: libapparmor, libgcrypt, libtasn1, llvm, lzip

<details><summary>Partial log (click to expand)</summary><p>

```
/build/llvm/lib/CodeGen/WinEHPrepare.cpp:1017:6: warning: 'void {anonymous}::WinEHPrepare::verifyPreparedFunclets(llvm::Function&)' defined but not used [-Wunused-function]
 void WinEHPrepare::verifyPreparedFunclets(Function &F) {
      ^~~~~~~~~~~~
[ 17%] Building CXX object lib/CodeGen/CMakeFiles/LLVMCodeGen.dir/XRayInstrumentation.cpp.o
[ 17%] Linking CXX static library ../libLLVMCodeGen.a
[ 17%] Built target LLVMCodeGen
Scanning dependencies of target LLVMSelectionDAG
[ 17%] Building CXX object lib/CodeGen/SelectionDAG/CMakeFiles/LLVMSelectionDAG.dir/DAGCombiner.cpp.o
building of '/nix/store/8z8zqhifpbfqq8iqni5l9m66fqs6ram1-llvm-5.0.2.drv' timed out after 3600 seconds
error: build of '/nix/store/8z8zqhifpbfqq8iqni5l9m66fqs6ram1-llvm-5.0.2.drv' failed
```
</p></details>

--- a/pkgs/development/compilers/llvm/5/llvm.nix
+++ b/pkgs/development/compilers/llvm/5/llvm.nix
@@ -79,8 +79,8 @@ in stdenv.mkDerivation (rec {
     "-DLLVM_ENABLE_RTTI=ON"
 
     "-DLLVM_HOST_TRIPLE=${stdenv.hostPlatform.config}"
-    "-DLLVM_DEFAULT_TARGET_TRIPLE=${stdenv.targetPlatform.config}"
-    "-DTARGET_TRIPLE=${stdenv.targetPlatform.config}"
+    "-DLLVM_DEFAULT_TARGET_TRIPLE=${stdenv.hostPlatform.config}"
+    "-DTARGET_TRIPLE=${stdenv.hostPlatform.config}"
   ]
   ++ stdenv.lib.optional enableSharedLibraries
     "-DLLVM_LINK_LLVM_DYLIB=ON"
@@ -129,8 +129,7 @@ in stdenv.mkDerivation (rec {
     ln -s $lib/lib/libLLVM.dylib $lib/lib/libLLVM-${release_version}.dylib
   '';
 
-  doCheck = stdenv.isLinux && (!stdenv.isi686)
-            && (stdenv.hostPlatform == stdenv.targetPlatform);
+  doCheck = stdenv.isLinux && (!stdenv.isi686);
 
   checkTarget = "check-all";
 

--- a/pkgs/development/compilers/llvm/6/llvm.nix
+++ b/pkgs/development/compilers/llvm/6/llvm.nix
@@ -75,8 +75,8 @@ in stdenv.mkDerivation (rec {
     "-DLLVM_ENABLE_RTTI=ON"
 
     "-DLLVM_HOST_TRIPLE=${stdenv.hostPlatform.config}"
-    "-DLLVM_DEFAULT_TARGET_TRIPLE=${stdenv.targetPlatform.config}"
-    "-DTARGET_TRIPLE=${stdenv.targetPlatform.config}"
+    "-DLLVM_DEFAULT_TARGET_TRIPLE=${stdenv.hostPlatform.config}"
+    "-DTARGET_TRIPLE=${stdenv.hostPlatform.config}"
 
     "-DLLVM_ENABLE_DUMP=ON"
   ]
@@ -130,8 +130,7 @@ in stdenv.mkDerivation (rec {
     ln -s $lib/lib/libLLVM.dylib $lib/lib/libLLVM-${release_version}.dylib
   '';
 
-  doCheck = stdenv.isLinux && (!stdenv.isi686)
-            && (stdenv.hostPlatform == stdenv.targetPlatform);
+  doCheck = stdenv.isLinux && (!stdenv.isi686);
 
   checkTarget = "check-all";
 

